### ISSUE TITLE
The API client lacks an interface for formData

### ISSUE THREAD

**Issue by hanspagel** [scalar/scalar#794]
Currently, there’s no UI to send form data with the API client.

Example:
```json
{
  "openapi": "3.1.0",
  "info": {
    "title": "Hello World",
    "version": "1.0.0"
  },
  "paths": {
    "/formData": {
      "post": {
        "parameters": [
          {
            "name": "id",
            "in": "formData",
            "schema": {
              "type": "number",
              "example": 1
            }
          },

          {
            "name": "name",
            "in": "formData",
            "schema": {
              "type": "string",
              "example": "foobar"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "Default Response"
          }
        }
      }
    }
  }
}
```

<img width="609" alt="Screenshot 2024-01-16 at 14 04 00" src="https://github.com/scalar/scalar/assets/1577992/c9ae784e-57cd-4790-bb03-7ce2a655dbb0">

**Comment by joelsonm** [comment]
I'm interested in this feature 👍

**Comment by marclave** [comment]
we should definitely leverage the the table inside of the Body section with the option to choose JSON | Form Data

@cameronrohani we have some things mocked up for this? or shall we mock them up : )

**Comment by zozobalogh0817** [comment]
Cannot wait to see this in Scalar 🎉

**Comment by hanspagel** [comment]
Solved #2302 :)
